Model/Checker/Preferences.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace SomethingDigital\UpgradeHelper\Model\Checker;
+
+use Magento\Framework\Autoload\AutoloaderRegistry;
+use Magento\Framework\ObjectManager\ConfigInterface as ObjectManagerConfig;
+
+class Preferences
+{
+    private $objectManagerConfig;
+
+    private $preferenceArray = [];
+
+    public function __construct(
+        ObjectManagerConfig $objectManagerConfig
+    ) {
+        $this->objectManagerConfig = $objectManagerConfig;
+        $this->autoloader = AutoloaderRegistry::getAutoloader();
```

Comment:
Looks like autoloader is missing as a private variable in this class.

Reply:
@ryandiamond23 good call. Check again please